101-backup-oms-monitoring/azuredeploy.json

```diff
@@ -2,9 +2,9 @@
     "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
     "contentVersion": "1.0.0.0",
     "parameters": {
-        "workspaceLocation": {
+        "location": {
             "type": "string",
-            "defaultValue": "",
+            "defaultValue": "[resourceGroup().location]",
```

Comment:
this is not a global resource so you need to remove the defaultValue and add allowedValues